contributing.md

```diff
@@ -49,7 +49,5 @@ yarn test:dts
 ```
 
 ### How to publish
-- Create a new branch
-- Run `yarn release:(major|minor|patch)` depending on the version you need
-- Make a pull request
-- After merging the pull request into the main branch and after successful tests, GitHub Action will publish a new version to the npm
\ No newline at end of file
+
+The library is automatically released and published to NPM on every push to the main branch if there are relevant changes. The workflow must be approved by one of the maintainers, first.
```

Comment:
Add please more information about release mechanism with the links to the documentation for all of used instruments